### ISSUE TITLE
NVDA 2020.1 stops narration of the slider

### ISSUE THREAD

**Issue by kolaps33** [nvaccess/nvda#11213]
# Steps to reproduce:
1. Open  the slider example
https://w3c.github.io/aria-practices/examples/slider/slider-1.html
2. Navigate to the slider with NVDA 
3. change slider value with down/up/right/left arrow keys

### Actual behavior:
NVDA doen't narrate anything

### Expected behavior:
NVDA narrates changed value on the slider

#### NVDA installed/portable/running from source:
installed NVDA on the PC

#### NVDA version:
2020.1

#### Windows version:
10.0.18363

#### Name and version of other software in use when reproducing the issue:
this bug was reproducible 
with Chrome  83.0.4103.61 
as well with Firefox 76.0.1

#### Does the issue still occur after restarting your computer?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
works with portable nvda_2019.3.1

**Comment by Adriani90** [comment]
@kolaps33 could you please run the COM registration fixing tool and try to reproduce this again?
Also please try with last NVDA alpha. I cannot reproduce in alpha-20220,1c4519a6.
You can download the last alpha version here:
https://www.nvaccess.org/files/nvda/snapshots/

**Comment by bhavyashah** [comment]
I am unable to replicate this issue with NVDA version 2020.2 Beta 1 on both Firefox version 77.0.1 and Brave browser (based on Chrome) version 83.1.10.93. NVDA reports the updated value of all three sliders - red, blue and green examples - as expected for all four arrow keys.

**Comment by Adriani90** [comment]
Thanks for the update @bhavyashah. I am closing this issue as works for me.
